plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/IcebergGlueCatalogConfig.java

```diff
@@ -19,7 +19,7 @@
 public class IcebergGlueCatalogConfig
 {
     private boolean cacheTableMetadata = true;
-    private boolean skipArchive;
+    private boolean skipArchive = true;
```

Comment:
Add some rationale explaining the `why` for the default value (unless it's obvious)

Reply:
Updated the commit comment.